src/main/webapp/style.css

```diff
@@ -10,15 +10,26 @@ body {
 
 .heading-font {
   font-family: 'Cinzel', serif;
-  color: rgb(237, 243, 121);
+  color: rgba(0,0,0, 0.4);
   font-size: 100px;
   text-align: center;
   
 }
 
 .body-font {
   font-family: 'Cinzel', serif;
-  color: whitesmoke;
-  font-size: 75px;
+  background-color: rgb(0,0,0); /* Fallback color */
+  background-color: rgba(0,0,0, 0.4); /* Black w/opacity/see-through */
+  color: white;
+  font-weight: normal;
+  font-size: 35px;
+  border: 10px solid #f1f1f1;
+  position: absolute; /* Stay fixed */
```

Comment:
Depending on how much more of the fiddly position you have to do, definitely consider bootstrap at a later point:

it should be pretty easy to try out: 
https://getbootstrap.com/docs/4.5/getting-started/introduction/

And it seems particularly good for really row-based stuff and navbars: https://getbootstrap.com/docs/4.5/examples/